src/components/Molecules/SimpleSchoolLookup/SimpleSchoolLookup.js

```diff
@@ -41,23 +41,28 @@ const schoolToString = school => `${school.name}, ${school.post_code}`;
  * @returns {JSX.Element}
  * @constructor
  */
-const SimpleSchoolLookup = ({ onSelect, ...rest }) => (
+const SimpleSchoolLookup = ({ onSelect, noResultsMessage, ...rest }) => (
   <Lookup
     name="school_lookup"
     label="Enter the name or postcode of your organisation"
     placeholder="Enter name or postcode..."
     buttonText="Find school"
     dropdownInstruction="Please select an organisation from the list below"
-    noResultsMessage="Sorry, could not find anything matching your search"
+    noResultsMessage={noResultsMessage}
     lookupHandler={schoolFetcher}
     mapOptionToString={schoolToString}
     onSelect={onSelect}
     {...rest}
   />
 );
 
+SimpleSchoolLookup.defaultProps = {
+  noResultsMessage: 'Sorry, we could not find anything matching your search; please use the manual entry option.'
+};
+
 SimpleSchoolLookup.propTypes = {
-  onSelect: PropTypes.func.isRequired
+  onSelect: PropTypes.func.isRequired,
+  noResultsMessage: PropTypes.string
```

Comment:
Do you need to mark this as required too, or provide a default somewhere?

Reply:
Nah; it's only 'required' by the component if there's _not_ a default provided, which I've done :)

Reply:
Oh I see, the thing just above this where you've put the message is `defaultProps`. Thanks, all good!